### PR TITLE
MariaDB/MySQL Global settings

### PR DESCRIPTION
**Pull request by riegercloud**
Added transaction_isolation and binlog_format to the database server configuration with an example of an entire /etc/mysql/my.cnf

### CODE REVIEW

File: admin_manual/configuration_database/linux_database_configuration.rst [hunk 1]
```diff
@@ -41,6 +41,57 @@ Configuring a MySQL or MariaDB database
...
+* The transaction isolation level is set to "READ-COMMITED" in your MariaDB server configuration :file:`/etc/mysql/my.cnf` to persist even after a restart of your database server.
+
+  Verify the **transaction_isolation** and **binlog_format**:
+
+::
+
+  [mysqld]
+  ...
+  transaction_isolation = **READ-COMMITTED**
+  binlog_format = **ROW**
```
Comment (MorrisJobke) [inline admin_manual/configuration_database/linux_database_configuration.rst]: Those `**` are not translated to bold text so I would drop it for now to not cause broken config. Same goes for the ones below. Beside that it looks good 👍

Comment (riegercloud) [reply admin_manual/configuration_database/linux_database_configuration.rst]: Great! Thank you!

Comment (MorrisJobke) [reply admin_manual/configuration_database/linux_database_configuration.rst]: Could you do those changes?

Comment (riegercloud) [reply admin_manual/configuration_database/linux_database_configuration.rst]: done.

**Comment by MorrisJobke** [comment]
stable12 f455f491 190eed0f 9c835292
stable13 ab6ee7f2 59a05717 80447bc8

**Pull request merged**